packages/cuid/cuid.0.1/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+maintainer: "Marco Aurélio da Silva <marcoonroad@gmail.com>"
+authors: "Marco Aurélio da Silva <marcoonroad@gmail.com>"
+homepage: "https://github.com/marcoonroad/ocaml-cuid"
+bug-reports: "https://github.com/marcoonroad/ocaml-cuid/issues"
+license: "MIT"
+dev-repo: "git://github.com/marcoonroad/ocaml-cuid.git"
+build: [
+  ["jbuilder" "subst"] {pinned}
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+install: [make "install"]
+build-test: [make "test"]
+remove: ["ocamlfind" "remove" "cuid"]
```

Comment:
If tou use dune/jbuilder you don’t need the install and remove steps, opam will use the .install files that it generates to do the right thing